media-gfx/openexr_viewers/openexr_viewers-2.2.1.ebuild

```diff
@@ -51,5 +57,5 @@ src_install() {
 		docdir=/usr/share/doc/${PF}/pdf \
```

Comment:
please also fix, same issue again, some stupid `Makefile.am` is defining `docdir = blabla` somewhere, given that you're eautoreconf'ing already, fixing this shouldn't be a massive effort